modules/standard/IO.chpl

```diff
@@ -415,6 +415,28 @@ Syntax in Detail" section below.
 In this file, we use "integral" to refer to the Chapel types int or uint and
 "floating-point" to refer to real, imaginary, or complex, of any bit width.
 
+Formatted I/O for C Programmers
++++++++++++++++++++++++++++++++
+
+========  ===========  ==========================================
+C         Chapel       Meaning
+========  ===========  ==========================================
+%i        %i           an integer in decimal
+%d        %i           an integer in decimal
+%x        %xu          an unsigned number in hexadecimal
+%g        %r           real number in exponential or decimal if compact
+%.2g      %2.r         real number with 2 significant digits
+%f        %dr          real number always in decimal
+%.3f      %.3dr        real number in decimal with 3 digits after ``.``
+%e        %er          real number always in exponential
+%.3e      %.3er        real number in exponential with 3 digits after ``.``
```

Comment:
The above e/f/\g forms also can take a number prior to the '.', right?  (as can %d and such, I think?)  Does it make sense to have an example of one of those as well/instead?  (e.g., "%13.5g")